std/php7/ArrayAccess.hx

```diff
@@ -0,0 +1,13 @@
+package php7;
+
+/**
+	Native PHP interface.
+	@see http://php.net/manual/en/class.arrayaccess.php
+**/
+@:native('ArrayAccess')
+extern interface ArrayAccess<K,V> {
```

Comment:
Yes, we can add `extends StdTypes.ArrayAccess<V>`. And it will work.
But is it a good idea? Here is a quote from Haxe `ArrayAccess` doc:
> This interface should be used for externs only

Perhaps we should leave it for user to decide, so that he is 100% aware how things works and what is happening in his app.